src/backend/web/routes/index.js

```diff
@@ -44,7 +44,14 @@ router.use('/legacy', express.static(path.join(__dirname, '../planet/static')));
 if (process.env.NODE_ENV === 'development') {
   if (process.env.PROXY_FRONTEND) {
     // Allow proxying the Next dev server through our backend if PROXY_FRONTEND=1 is set in env
-    router.use('/', createProxyMiddleware({ target: 'http://localhost:8000', changeOrigin: true }));
+    router.use(
+      '/',
+      createProxyMiddleware({
+        target: 'http://localhost:8000',
+        changeOrigin: true,
+        logLevel: 'silent',
```

Comment:
@humphd I have a question, why didn't we set createProxyMiddleware logLevel to silent? Is it critical for developers?

Reply:
No, we don't use the logs in any way.  We use this proxy only in some cases in development, not for production.